deploy/FlapperInit.sol

```diff
@@ -143,22 +150,41 @@ library FlapperInit {
         dss.chainlog.setAddress(clKey, wrapper_);
     }
 
+    function initFarm(
```

Comment:
The intention for this is to run on top of another initiation function for the farm?
Otherwise we might need to check more stuff like the staking token.
So maybe it should be mentioned that this is not really a regular init function (we also don't have a deploy function for the farm).

Reply:
Hmm I think that should be part of the init of Lockstake (check that every farm that is added, the staking token is `lsmkr`). In this side we only care about the rewards part as we have contact.
I agree it is a weird case and it goes out of the standard. For this specific case what I picture is you use the deploy script from the endgame-toolkit repo itself (or not, it doesn't really matter, as it is part of the spell review process to check a new contract is relied to the pause proxy and denied from the deployer), then the initiation happens in this repo init script verifying the rewards token is the one that the splitter will be sending, and the Lockstake makes sure that the farm/s being added are using the corresponding staking token.

Reply:
Yeah ok, was referring more in a general sense.